policy/runtime/runtime.go

```diff
@@ -134,7 +134,7 @@ func (t *Template) Name() string {
 // Validate checks the content of an instance to ensure it conforms with the validation rules
 // present within the template, if any.
 func (t *Template) Validate(src *model.Source, inst *model.Instance) *cel.Issues {
-	if t.validator == nil {
+	if t == nil || t.validator == nil {
```

Comment:
Why would the template itself be `nil`? Or is this just general defensive coding?

Reply:
It could happen here https://github.com/google/cel-policy-templates-go/blob/master/policy/compiler/compiler.go#L328-L336. Sorry it's been a while, I forgot the exact scenario.

Reply:
Thanks, that's good enough for me.